reports/example.md

```diff
@@ -5,40 +5,50 @@ layout: report
 permalink: /GSoC/2017/StatusReport/<name>
 student: <name>
 organisation: coala
-organisation_link : https://coala.io
+organisation_link: https://coala.io
 project: <project title>
 project_link: <project link>
+report_year: <year of completing GSoC>
 tarball: <tarball of patches' link>
+sha256sum: <sha256sum of tarball>
 mentors: >
- [mentor1]<link1> & [mentor2]<link2>
+  [mentor1]<link1> & [mentor2]<link2>
 phase:
- - Bonding : <bonding milstone link>
- - Phase 1 : <phase1 milsetone link>
- - Phase 2 : <phase2 milsetone link>
- - Phase 3 : <phase3 milsetone link>
+  - Bonding : <bonding milestone link>
+  - Phase 1 : <phase1 milsetone link>
+  - Phase 2 : <phase2 milsetone link>
+  - Phase 3 : <phase3 milsetone link>
```

Comment:
`milsetone` together with above two is also a typo :P

Reply:
Holy smokes!